File: JavaPrograms/Array/src/array/ArrayDeletion.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package array;

/**
 *program name: To delete a number in the given array
 * @author harid
 */
import java.util.Scanner;
public class ArrayDeletion {
public static void main(String[] args) {
Scanner input=new Scanner(System.in);
int arr[]=new int[6];
int size=5,i;
System.out.println("enter the elements of the array");       
for(i=0;i<size;i++)
{
    arr[i]=input.nextInt();
    
}
    System.out.println("enter the position");
    int position=input.nextInt();
    for(i=position;i<size;i++)
    {
        arr[i]=arr[i+1];
    }    
    System.out.println("after deletion");
    for(i=0;i<size;i++)
    {
        System.out.println(arr[i]);   
    }
    }
   
    
}

File: CPP_Programs/array_deletion.cpp
//program name : Array Deleting a value in the given array
#include<iostream>
using namespace std;
int main()
{
	int a[10],i,size,posi;
	cout<<"enter the size of a array\n";
	cin>>size;
	cout<<"\nenter the elements in the array\n";
	for(i=0;i<size;i++)
	{
	cin>>a[i];
	}
	cout<<"before deletion\n";
	for(i=0;i<size;i++)
	{
	cout<<endl<<a[i];
	}
	cout<<"\nenter the position in which the element to be deleted\n";
	cin>>posi;
	for(i=posi;i<size;i++)
	{
	a[i]=a[i+1];	
	}
	cout<<"after deletion";
	for(i=0;i<size-1;i++)
	{
 cout<<endl<<a[i];
	}
}
